### PR TITLE
Schedule and speakers announced

### PR DESCRIPTION
**Pull request by swapnilogale**

### CODE REVIEW

File: docs/_data/config-australia-2018.yaml [hunk 1]
```diff
@@ -65,7 +65,7 @@ flaglanding: False
 flagspeakersannounced: False
 flagcfp: False
 flaghassponsors: True
-flaghaschedule: True
+flaghasschedule: True
```
Comment (plaindocs) [inline docs/_data/config-australia-2018.yaml]: :-D :+1:

**Review by plaindocs — APPROVED** [review]
Looks ok to me.

**Comment by plaindocs** [comment]
Also, could you make sure your title underlines are as long as the titles:

```
/www/docs/conf/australia/2018/schedule.rst:49: WARNING: Title underline too short.

Thursday Night Social
~~~~~~~~~~~~~~~~~~~
/www/docs/conf/australia/2018/schedule.rst:49: WARNING: Title underline too short.

Thursday Night Social
~~~~~~~~~~~~~~~~~~~
/www/docs/conf/australia/2018/schedule.rst:80: WARNING: Title underline too short.

Workshop 1: Let's Create a Style Guide
~~~~~~~~~~~~~
/www/docs/conf/australia/2018/schedule.rst:80: WARNING: Title underline too short.

Workshop 1: Let's Create a Style Guide
~~~~~~~~~~~~~
/www/docs/conf/australia/2018/schedule.rst:87: WARNING: Title underline too short.

Workshop 2: Static Site Generators, What, Why and How
~~~~~~~~~~~~~
/www/docs/conf/australia/2018/schedule.rst:87: WARNING: Title underline too short.

Workshop 2: Static Site Generators, What, Why and How
~~~~~~~~~~~~~
/www/docs/conf/australia/2018/team.rst:14: WARNING: Title underline too short.

Swapnil Ogale
~~~~~~~~~~~
/www/docs/conf/australia/2018/team.rst:29: WARNING: Title underline too short.

Russell Dickenson
~~~~~~~~~~~~~
/www/docs/conf/australia/2018/team.rst:29: WARNING: Title underline too short.

Russell Dickenson
~~~~~~~~~~~~~
```

**Comment by plaindocs** [comment]
Just looking at what is breaking this

**Comment by swapnilogale** [comment]
Thanks @plaindocs - I am just trying out one speaker at this stage to make sure it is working fine, before I load all the others

**Comment by swapnilogale** [comment]
Fantastic @plaindocs - thanks. I am going to load all the speakers and then get someone to review it before we merge it in 👍

**Comment by swapnilogale** [comment]
@sarahmaddox can you quickly review this? I will publish this once you do and are ok. 
The main changes are https://deploy-preview-654--dentist-canary-64055.netlify.com/conf/australia/2018/speakers/
 and https://deploy-preview-654--dentist-canary-64055.netlify.com/conf/australia/2018/schedule/

File: docs/_data/au-2018-day-2.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  Time: '14:30'
+- Duration: '10'
+  Session: Switch Speakers
+  Time: '15:30'
+- Duration: '60'
+  Session: "Workshop - Static Site Generators, What, Why and How \u2013 Jessica Parsons"
+  Slug: jessica-parsons
+  Time: '15:40'
+- Duration: '15'
+  Session: Day 1 Wrap Up
```
Comment (sarahmaddox) [inline docs/_data/au-2018-day-2.yaml]: Day 2

File: docs/conf/australia/2018/schedule.rst [hunk 3]
```diff
@@ -46,43 +52,56 @@ The unconference sessions run in parallel with the workshop.
...
+* **When**: **5.30pm-8pm**
 
 
 Friday, 16th November 2018
 ----------------------------------------
 
-*Tea and coffee will be provided, as well as snacks and drinks all day.*
+*Tea, coffee and lunch will be provided, as well as snacks and drinks all day.*
 
 .. contents::
```
Comment (sarahmaddox) [inline docs/conf/australia/2018/schedule.rst]: I think we should remove the table of contents on this page. We don't have a corresponding table of contents for Thursday, and it seems overkill to list the sections of the page here.

**Review by sarahmaddox — APPROVED** [review]
Just 2 comments then LGTM!

**Comment by swapnilogale** [comment]
@plaindocs - I am happy with this now. I've edited pages Sarah mentions above.
Can you give the News, Schedule, Talks and Sponsors page a quick look and merge this PR? 

Thanks!

**Comment by plaindocs** [comment]
Thanks for fixing up the heading underlines @swapnilogale .

Fixed up a couple of minor things, the rest looks great!

**Pull request merged**